stdlib/logging/handlers.pyi

```diff
@@ -249,17 +250,21 @@ class HTTPHandler(Handler):
     if sys.version_info >= (3, 9):
         def getConnection(self, host: str, secure: bool) -> http.client.HTTPConnection: ...  # undocumented
 
+class _QueueLike(Protocol[_T]):
+    def get(self) -> _T: ...
+    def put_nowait(self, item: _T) -> None: ...
```

Comment:
We also need `task_done`, it's used in `QueueListener._monitor`.

Also, `QueueListener.dequeue` has comments suggesting it could be overridden to use something other than `.get()`. I think we can ignore that for typing purposes though: if you make a subclass you can provide different types yourself.

Reply:
Good point, I missed that.